docs/platform/howto/byoc/create-custom-cloud.rst

```diff
@@ -472,11 +472,19 @@ In the **Create custom cloud** wizard, proceed as follows:
    * Region
    * CIDR
 
-     Aiven needs CIDR for the `CIDR block of the VPC <https://docs.aws.amazon.com/vpc/latest/userguide/vpc-cidr-blocks.html>`_ that will be created in your AWS account.
+     **CIDR** represents an IP address range of the BYOC VPC. It needs to be specified in the IaC template so that a CIDR block of the VPC can be created in your AWS account while deploying the template.
```

Comment:
I think this could be phrased better: user gives us a single CIDR block via our API, and then we parametrize the template to create a VPC with that CIDR block, and then subnets in that VPC. "CIDR block of the VPC can be created" is not that clear to me.